requirements/testing.txt

```diff
@@ -11,3 +11,4 @@ pytest-cov==4.0.0
 setuptools==67.1.0
 tox==4.4.4
 mypy==0.991
+codespell==2.2.2
```

Comment:
Do we need this as explicit requirement when we only use it in per-commit?

Reply:
I wasn't sure if you track this way if the pre-commit requirements are up to date.